src/bio/bio_xstream.c

```diff
@@ -37,6 +38,9 @@
 #include "bio_internal.h"
 #include <daos_srv/smd.h>
 
+/* FIXME: remove it once SPDK being upgraded */
+void spdk_set_thread(struct spdk_thread *thread);
```

Comment:
(style)  externs should be avoided in .c files

Reply:
@NiuYawei should we ignore this warning?

Reply:
Yes, I think we should ignore the warning. This is a temporary workaround and it'll be removed once SPDK upgrade to later version which have this function exported.